haiku-misc/gadgeteer/gadgeteer-0.8.1.recipe

```diff
@@ -31,6 +32,7 @@ BUILD_PREREQUIRES="
 	makefile_engine
 	cmd:cc
 	cmd:make
+	devel:libyaml
```

Comment:
This should go to "BUILD_REQUIRES" :)

Reply:
@Begasus Whoops! That was what I meant :D